bool safety_ready       # sau khi resume release
bool safety_good        # he thong co the chay
bool muted_safety
bool operator_move

fx3_saf_protective_fault device_state
fx3_saf_mode_switch mode_switch
fx3_saf_zone zone
fx3_saf_protective_field laser_field
fx3_saf_protective_field camera_field